src/view/uniprotkb/components/VariationView.tsx

```diff
@@ -46,17 +53,120 @@ interface ChangeEvent extends Event {
   loadWebComponent('protvista-variation-adapter', ProtvistaVariationAdapter);
 
+  const [showEvidenceTagData, setShowEvidenceTagData] = useState(false);
+  const [selectedEvidenceData, setSelectedEvidenceData] = useState();
+  const [selectedReferences, setSelectedReferences] = useState();
+
+  const evidenceTagCallback = (
+    evidenceData: EvidenceData,
+    references: Evidence[] | undefined
+  ) => {
+    setSelectedEvidenceData(evidenceData);
+    setSelectedReferences(references);
+    setShowEvidenceTagData(true);
+  };
+
+  const columns = {
+    positions: {
+      label: 'Position(s)',
+      resolver: (d: ProtvistaVariant) =>
+        d.begin === d.end ? d.begin : `${d.begin}-${d.end}`,
+    },
+    change: {
+      label: 'Change',
+      resolver: (d: ProtvistaVariant) =>
+        `${d.wildType}>${d.alternativeSequence}`,
+    },
+    consequence: {
+      label: 'Consequence',
+      child: true,
+      resolver: (d: ProtvistaVariant) => d.consequenceType,
+    },
+    sift: {
+      label: 'SIFT prediction',
+      child: true,
+      resolver: (d: ProtvistaVariant) => `${d.siftPrediction} (${d.siftScore})`,
+    },
+    polyphen: {
+      label: 'Polyphen prediction',
+      child: true,
+      resolver: (d: ProtvistaVariant) =>
+        `${d.polyphenPrediction} (${d.polyphenScore})`,
+    },
+    description: {
+      label: 'Description',
+      resolver: (d: ProtvistaVariant) => {
+        if (!d.description) {
+          return html``;
+        }
+        const formatedDescription = formatVariantDescription(d.description);
+        return (
+          formatedDescription &&
+          formatedDescription.map(
+            descriptionLine =>
+              html`
+                <p>${descriptionLine}</p>
+              `
+          )
+        );
+      },
+    },
+    somaticStatus: {
+      label: 'Somatic',
+      child: true,
+      resolver: (d: ProtvistaVariant) => (d.somaticStatus === 1 ? 'Yes' : 'No'),
+    },
+    hasDisease: {
+      label: 'Disease association',
+      resolver: (d: ProtvistaVariant) =>
+        d.association && d.association.length > 0 ? 'Y' : 'N',
+    },
+    association: {
+      label: 'Disease association',
+      child: true,
+      resolver: (d: ProtvistaVariant) =>
+        d.association
+          ? d.association.map(association => {
+              return html`
+                <p>
+                  ${association.name}
+                  ${association.evidences &&
+                    UniProtProtvistaEvidenceTag(
+                      association.evidences.map(evidence => {
+                        return ({
+                          evidenceCode: evidence.code,
+                          source: evidence.source.name,
+                          id: evidence.source.id,
+                        } as unknown) as Evidence;
+                      }),
+                      evidenceTagCallback
+                    )}
+                </p>
+              `;
+            })
+          : '',
```

Comment:
There are several lines between the `?` and the `:` in this ternary expression. Could be a little easier to read if the null-case is returned immediately as you have done with the `description`'s resolver.